benchmark/Streamly/Benchmark/Data/Stream/Eliminate.hs

```diff
@@ -494,11 +531,12 @@ o_n_heap_elimination_foldl value =
         -- Left folds for building a structure are inherently non-streaming
         -- as the structure cannot be lazily consumed until fully built.
         [ benchIOSink value "foldl'/build/IO" foldl'Build
-        , benchIdentitySink value "foldl'/build/Identity" foldl'Build
         , benchIOSink value "foldlM'/build/IO" foldlM'Build
+        , benchIdentitySink value "foldl'/build/Identity" foldl'Build
```

Comment:
Why change the order? We had `foldl'` IO/Identity benchmarks together for comparison.